code/TemplateStudioForWinUICs/Templates/_comp/MT/Project.MenuBar/Views/ShellPage.xaml

```diff
@@ -26,8 +26,8 @@
                 <RowDefinition Height="*" />
             </Grid.RowDefinitions>
             <Grid Grid.Row="0" x:Name="AppTitleBar">
-                <Image Source="design\Assets\Logo\TemplateStudio_16.png" HorizontalAlignment="Left" Width="16" Height="16" Margin="8,0"/>
-                <TextBlock x:Name="AppTitleBarText" Text="App Title" VerticalAlignment="Center" TextWrapping="NoWrap" Style="{StaticResource CaptionTextBlockStyle}" Margin="28, 0, 0, 0"/>
+                <Image x:Name="TitleBarImage" Source="/Assets/WindowIcon.png" HorizontalAlignment="Left" Width="16" Height="16" Margin="8,0"/>
```

Comment:
We don't use this name anywhere in code behind so let's remove it for now. Otherwise I'd call it AppTitleBarImage to align with AppTitleBar above.